chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -29,8 +29,13 @@ const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, state:
         if (conversationDetails?.canRenderPostChat?.toLowerCase() === Constants.false) {
             // If ended by customer, just close chat
             if (state?.appStates?.conversationEndedBy === ConversationEndEntity.Customer) {
+                TelemetryHelper.logSDKEvent(LogLevel.INFO, {
+                    Event: TelemetryEvent.PrepareEndChat,
+                    Description: "Conversation ended by customer. Post chat not configured or should not show."
```

Comment:
Should these `Description` be refactored an external file for `ConversationEndReasonDescription`?